runtests.py

```diff
@@ -987,7 +998,8 @@ def filter_test_suite(test_suite, selector):
 
 class CythonCompileTestCase(unittest.TestCase):
     def __init__(self, test_directory, workdir, module, module_path, tags, language='c', preparse='id',
-                 expect_errors=False, expect_warnings=False, annotate=False, cleanup_workdir=True,
+                 expect_errors=False, expect_warnings=False, expect_perf_hints=False,
```

Comment:
I wonder if it's time to clean this up and pass a single `expect_log=('warnings', 'errors')` option around (name is up for debate…). We're really carrying around a lot of options by now, and I feel bad every time I add one to the list.

Doesn't have to be in this PR, though. Adding this one doesn't make it much worse.